templates/nextcloud28-entrypoint.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+
+chown -R www-data:www-data /var/www/html
+chmod -R 775 /var/www/html
+
+# Let /entrypoint.sh not run apache
+sed -i 's/exec "$@"//g' /entrypoint.sh
+
+# create file to override config
+cat > /var/www/html/config/liquid.override.config.php << 'DELIM'
+<?php
+$CONFIG = array (
+  'dbhost' =>  getenv('MYSQL_HOST'),
+  'trusted_domains' => [
+                    "localhost",
+                    getenv('NEXTCLOUD_HOST'),
+                    getenv('LIQUID_BASE_IP')
+],
+  'dbname' => getenv('MYSQL_DATABASE'),
+  'dbpassword' => getenv('MYSQL_PASSWORD'),
+);
+DELIM
+
+# Launch canonical entrypoint
+/entrypoint.sh apache2-foreground
+
+unzip /apps-to-install/sociallogin.zip -d /apps-to-install
+mkdir -p /var/www/html/custom_apps/sociallogin
+mv /apps-to-install/nextcloud-social-login-*/* /var/www/html/custom_apps/sociallogin
```

Comment:
check if already installed and don't install again, to avoid doing 200MB of I/O every container start -- better yet do this only once in initial installation step